backend/lcfs/web/api/organization/views.py

```diff
@@ -333,3 +363,70 @@ async def get_users_for_organization(
     except Exception as e:
         logger.error(f"Error getting users for organization: {str(e)}")
         raise HTTPException(status_code=500, detail=f"Internal Server Error")
+
+
+@router.get("/{organization_id}/transactions/", response_model=Transactions)
+async def get_transactions_for_organization(
+    organization_id: int,
+    db: AsyncSession = Depends(get_async_db),
+    pagination: PaginationRequestSchema = Body(..., embed=False),
+    response: Response = None,
+):
+    try:
+        offset = 0 if (pagination.page < 1) else (pagination.page - 1) * pagination.size
+        limit = pagination.size
+
+        query = (
```

Comment:
Should we move the queries to session.py?

Reply:
Thanks for the comment. I think these changes are from Kevin's branch. I'll try to rebase mine with his after his branch is merged.

Reply:
No problem! Thanks for letting me know, Prashanth!